repo/packages/E/elasticsearch/1/config.json

```diff
@@ -0,0 +1,155 @@
+          "type":"string"
+        },
+        "zookeeper-timeout":{
+          "default":"20000",
+          "description":"The timeout for connecting to zookeeper (ms).",
+          "type":"string"
+        },
+        "port":{
+          "default":"31105",
+          "description":"The port number of the ES-Mesos management front-end. Defaults to 8080.",
+          "type":"string"
+        },
+        "force-pull-image":{
+          "default":false,
+          "description":"Forces docker to re-pull the image.",
+          "type":"boolean"
+        },
+        "executor":{
+          "properties":{
+            "ram":{
+              "default":"2048",
+              "description":"The amount of ram that Mesos shall give give to the executors.",
+              "type":"string"
+            },
+            "cpu":{
+              "default":"1.0",
+              "description":"The amount of CPU resource to allocate to the elasticsearch instance.",
+              "type":"string"
+            },
+            "disk":{
+              "default":"1024",
+              "description":"The amount of Disk resource to allocate to the elasticsearch instance (MB).",
+              "type":"string"
+            },
+            "instances":{
+              "default":"3",
+              "description":"The number of elasticsearch nodes to be instantiated. It is recommended that you provide an odd number, to prevent split brain issues.",
+              "type":"string"
+            },
+            "cluster-name":{
+              "default":"mesos-ha",
+              "description":"Name of the elasticsearch cluster",
+              "type":"string"
+            },
+            "settings-location":{
+              "default":"",
+              "description":"URI to ES yml settings file. If file is copied to all slaves, the file must be in /tmp/config. E.g. 'file:/tmp/config/elasticsearch.yml', 'http://webserver.com/elasticsearch.yml'",
+              "type":"string"
+            },
+            "health-delay":{
+              "default":"3000",
+              "description":"The delay between executor healthcheck requests (ms).",
+              "type":"string"
+            },
+            "image":{
+              "default":"mesos/elasticsearch-executor:0.3.0",
+              "description":"The docker executor image to use.",
+              "type":"string"
+            },
+            "name":{
+              "default":"elasticsearch-executor",
+              "description":"The name given to the executor task.",
+              "type":"string"
+            },
+            "timeout":{
+              "default":"60000",
+              "description":"The maximum executor healthcheck timeout (ms). Must be greater than elasticsearch.executor.healthdelay. Will start new executor after this length of time.",
+              "type":"string"
+            }
+          },
+          "required":[
```

Comment:
Ah, another problem. Aparrently the required parameter, doesn't actually mean this parameter is required by the user. It means it is required by the config parsing frameworks. 
I didn't really understand this, but basically it means that all parameters need to be in their respective required array. E.g. timeout should be here, etc.